packages/core-contracts/contracts/mocks/proxy/Bricker.sol

```diff
@@ -0,0 +1,43 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "../../proxy/UniversalProxyImplementation.sol";
+
+contract Bricker is UniversalProxyImplementation {
```

Comment:
This contracts test one case that I found not covered originally that is a contract that adheres to the paradigm but was poorly (or intentionality bad) developed.